### PR TITLE
frontend: Fix some console warnings

### PR DESCRIPTION
**Pull request by farodin91**
Part of #1612

@illume Would you like to look into this?

### CODE REVIEW

**Comment by illume** [comment]
Thank you very much @farodin91 
I will have a look at this later today.

**Review by illume — APPROVED** [review]
LGTM. 

I'm not sure about removing the backwards compatibility code though, and would like to know your thinking behind it? Is it causing log messages to appear when not using the deprecated method?

File: frontend/src/components/priorityClass/__snapshots__/priorityClassDetails.stories.storyshot [hunk 1]
```diff
@@ -72,20 +72,7 @@ exports[`Storyshots PriorityClass/PriorityClassDetailsView Default 1`] = `
               />
               <div
                 class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
-              >
```
Comment (illume) [inline frontend/src/components/priorityClass/__snapshots__/priorityClassDetails.stories.storyshot:75]: I wonder why this button isn't rendered now?

Comment (farodin91) [reply frontend/src/components/priorityClass/__snapshots__/priorityClassDetails.stories.storyshot:75]: I think auth check didn't work before. Now, it falls back, I would say. I didn't found any test which is already testing auth.

File: frontend/src/components/common/Resource/Resource.tsx [hunk 2]
```diff
@@ -244,19 +241,6 @@ export function DetailsGrid(props: DetailsGridProps) {
     });
   }
 
-  // Other sections
-  if (!!sectionsFunc) {
```
[the remark this answers is not in the document]
Comment (farodin91) [reply frontend/src/components/common/Resource/Resource.tsx:248]: I didn't found any use. I can bring it back.

File: frontend/src/components/priorityClass/__snapshots__/priorityClassDetails.stories.storyshot (hunk 1 above)
Comment (farodin91) [reply frontend/src/components/priorityClass/__snapshots__/priorityClassDetails.stories.storyshot:75]: @illume Do you know how to test or have an example to test AuthVisible?

Comment (farodin91) [reply frontend/src/components/priorityClass/__snapshots__/priorityClassDetails.stories.storyshot:75]: now edit is see able.

**Comment by farodin91** [comment]
I reverted the removal of sectionFunc

**Comment by illume** [comment]
Thanks @farodin91

**Pull request merged**